src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -406,13 +408,32 @@ export class SearchInterface extends RootComponent implements IComponentBindings
      * [Query Parameters - maximumAge](https://developers.coveo.com/x/iwEv#QueryParameters-maximumAge)).
      */
     maximumAge: ComponentOptions.buildNumberOption(),
-
     /**
      * Specifies the search page you wish to navigate to when instantiating a standalone search box interface.
      *
      * Default value is `undefined`, which means that the search interface does not redirect.
      */
-    searchPageUri: ComponentOptions.buildStringOption()
+    searchPageUri: ComponentOptions.buildStringOption(),
+    /**
+     * Specifies the search interface width that should be considered "medium" size, in pixel.
+     *
+     * This is at this breakpoint that, for examples, facets and filters in the "coveo-facet-column" will switch to display inside a dropdown above the result list instead of next to the result list.
+     *
+     * This also implies that {@link SearchInterface.options.enableAutomaticResponsiveMode} is set to `true`.
+     *
+     * The default value is `800`.
+     */
+    responsiveMediumBreakpoint: ComponentOptions.buildNumberOption({ defaultValue: MEDIUM_SCREEN_WIDTH }),
+    /**
+     * Specifies the search interface width that should be considered "small" size, in pixel.
+     *
+     * This is at this breakpoint that, for examples, some layouts in the result list will get disabled as they are not suited to display properly at smaller screen size.
```

Comment:
Suggestion:

When the width of the window/device that displays the search page reaches or falls short of this threshold, the search page layout will change so that, for instance, some result list layouts which are not suited for being rendered on a small screen/area will be disabled.